README.md

```diff
@@ -1,16 +1,18 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Title
+# Leader Board: Hit the API!
 
-> Project headline description
+> This simple project aims to connect to an API to fetch score data and display it on a LeaderBoard box, as well as provide the tool to submit a new score.
 
 ![Preview](./preview.gif))
 
```

Comment:
- [ ] This preview does not correspond with the output of your project. When I try to submit a score, the leaderboard doesn't update. The style is also not the same as the wireframe.

![image](https://user-images.githubusercontent.com/16058470/151435732-66cd34d7-b8e5-4d7d-b7e8-f8a7671661d0.png)

Reply:
Hi @ElijahTobs. Sorry, there's seems to be an issue with your webpack, may I ask you how did you set up the project locally?